api/src/Controller/Event/PostEvent.php

```diff
@@ -3,6 +3,51 @@
     require_module 'standard';
 .*/
 
+/**
+ *  @OA\Post(
+ *      tags={"events"},
+ *      path="/event",
+ *      summary="Adds a new event",
+ *      @OA\RequestBody(
+ *          @OA\MediaType(
+ *              mediaType="multipart/form-data",
```

Comment:
We should decide if, since we're publishing this now (not that anyone but us is using it yet), if we need to version things when we start shifting these all to JSON (and start shifting the parameter cases as we discussed)